auto_changelog/__main__.py

```diff
@@ -5,10 +5,22 @@
 import click
 
 from auto_changelog import generate_changelog
-from auto_changelog.presenter import MarkdownPresenter
+from auto_changelog.presenter import MarkdownPresenter, default_template
 from auto_changelog.repository import GitRepository
 
 
+def validate_template(ctx, param, value):
+
+    # Check if an embedded template is passed in parameter
+    if value in default_template:
+        return value
+    # Check if the custom template is a jinja2 file
+    elif value.endswith(".jinja2"):
```

Comment:
Hint: This is against the Unix convention of checking file content instead of a name.